src/credential_provider/ClientCredentialProvider.ts

```diff
@@ -0,0 +1,66 @@
+import CredentialProvider from "./CredentialProvider";
+import TokenManager from "../http/bearer_token/TokenManager";
+import AuthStrategy from "../auth_strategy/AuthStrategy";
+import ApiTokenManager from "../http/bearer_token/ApiTokenManager";
+import TokenAuthStrategy from "../auth_strategy/TokenAuthStrategy";
+
+class ClientCredentialProvider extends CredentialProvider {
```

Comment:
You need another Credential provider for Orgs API

Reply:
Added Orgs credential provider [here](https://github.com/twilio/twilio-node/pull/1049/files#diff-2eb25eb6042f942d4341f8b8c56dbeaeeba9660a513c7f5a936ad1a2a1c8a364)